tracer/src/Datadog.Trace/Ci/Tags/BuildTags.cs

```diff
@@ -77,6 +77,7 @@ internal static class BuildTags
 
         /// <summary>
         /// Error message
+        /// TODO: check with Tony, but I assume we shouldn't change the CI app spans
```

Comment:
Interesting 🤔 I'm just dropping a comment to make it easier to spot later

Reply:
You can change it, I talked with CI Visibility backend folks and we also have the same kind of remapper.

Reply:
🙇